Flaky test test_memory_profile_in_running_events

Sometimes this test can fail, especially on weaker hardware
```
FAILED tests/unit_tests/forward_model_runner/test_job.py::test_memory_profile_in_running_events - AssertionError: Memory increased too sharply, missing a measurement? Got 
```
https://github.com/equinor/ert/actions/runs/10193753149/job/28198962231

--- a/tests/unit_tests/forward_model_runner/test_job.py
+++ b/tests/unit_tests/forward_model_runner/test_job.py
@@ -132,9 +132,9 @@ def test_memory_profile_in_running_events():
     ).all(), f"Emitted memory usage not increasing, got {emitted_rss_values[:-3]=}"
 
     assert (
-        np.diff(np.array(emitted_rss_values[3:])).max() < 3 * 1024 * 1024
+        np.diff(np.array(emitted_rss_values[7:])).max() < 3 * 1024 * 1024
         # Avoid the first steps, which includes the Python interpreters memory usage
-    ), f"Memory increased too sharply, missing a measurement? Got {emitted_rss_values[3:]=}"
+    ), f"Memory increased too sharply, missing a measurement? Got {emitted_rss_values[7:]=}"
 
     if sys.platform.startswith("darwin"):
         # No oom_score on MacOS

